README.md

```diff
@@ -436,7 +436,7 @@ Where to discover new tools and discuss about existing ones.
 * [Production-Ready Applied Deep Learning](https://www.packtpub.com/product/production-ready-applied-deep-learning/9781803243665) (Packt)
 * [Reliable Machine Learning](https://www.oreilly.com/library/view/reliable-machine-learning/9781098106218) (O'Reilly)
 * [The Machine Learning Solutions Architect Handbook](https://www.packtpub.com/product/the-machine-learning-solutions-architect-handbook/9781801072168) (Packt)
-
+* [MLOps Lifecycle Toolkit](https://www.oreilly.com/library/view/implementing-mlops-in/9781098136574/) (Apress)
```

Comment:
The link points to the `Implementing MLOps in the Enterprise` book which is already in this list.
Could you please set the correct link and add the `MLOps Lifecycle Toolkit` in alphabetical order?

Reply:
@kelvins updated the link and added in alphabetical order.